WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.m

```diff
@@ -947,7 +942,9 @@ - (void)trashCommentWithBlock:(NotificationBlock *)block
 
 - (void)editReplyWithBlock:(NotificationBlock *)block
 {
-    EditReplyViewController *editViewController     = [EditReplyViewController newEditViewController];
+    BOOL shouldAddSuggestionView = [[SuggestionService sharedInstance] shouldShowSuggestionsForSiteID:self.note.metaSiteID];
+    NSNumber *siteID = shouldAddSuggestionView ? self.note.metaSiteID : nil;
+    EditReplyViewController *editViewController     = [EditReplyViewController newReplyViewControllerForSiteID:siteID];
```

Comment:
(Same annoying comment as above), maybe it'd be cleaner to let `EditReplyTextViewController` determine if it should render suggestions (or perhaps sending a boolean enabling / disabling that)